hoot-josm/pom.xml

```diff
@@ -0,0 +1,101 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 
+	http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>hoot.core.josm</groupId>
+	<artifactId>hoot-josm</artifactId>
+	<packaging>jar</packaging>
+	<version>0.0.1</version>
+	<name>hoot-josm</name>
+	<url>http://hootenanny.org</url>
+
+	<properties>
+		<jdk.version>1.8</jdk.version>
+                <josm.artifactId>josm</josm.artifactId>
+		<!-- josm.artifactId>me-josm</josm.artifactId -->
+		<josm.version>15628</josm.version>
+		<!-- josm.version>4.4.9</josm.version -->
+	</properties>
+
+  <repositories>
+    <repository>
+      <id>org.openstreetmap.josm</id>
```

Comment:
entry for public josm repo; I don't think our version of josm has a repo, so devops will have to figure out a workflow to use it (will open an issue).

Reply:
Have we told DevOps about this yet?

Reply:
Will make a ticket when the tests pass. The branch as it is now will function merged while pointed to public JOSM. The devops work would be for Tradecraft to get access to me-josm.